docs/docs/docs/overriding-behavior-for-case-classes.md

```diff
@@ -62,6 +62,7 @@ library:
 
 * [`CamelCase`](https://en.wikipedia.org/wiki/Camel_case) (examples: `camelCase`, `useMorePureconfig`);
 * [`SnakeCase`](https://en.wikipedia.org/wiki/Snake_case) (examples: `snake_case`, `use_more_pureconfig`);
+* ScreamingSnakeCase (examples: `SCREAMING_SNAKE_CASE`, `USE_MORE_PURECONFIG`);
```

Comment:
nit: can you make this monospace and link to https://en.wikipedia.org/wiki/Snake_case? I think it's still useful to link to it even if it's only mentioned in the History section.

```suggestion
* [`ScreamingSnakeCase`](https://en.wikipedia.org/wiki/Snake_case) (examples: `SCREAMING_SNAKE_CASE`, `USE_MORE_PURECONFIG`);
```